runcheckmigration

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -e
+checkmigrations --migration-path ./django_loci/migrations/ || exit 1
```

Comment:
Add an empty line at the end of a file. (Always add an empty line at the end of the file)

Reply:
done 👍